std/manual.md

```diff
@@ -1210,6 +1210,61 @@ console.log(result["/foo.ts"].map);
 We would expect the `enum` would be rewritten to an IIFE which constructs the
 enumerable, and the map to be defined.
 
+## TypeScript Compiler Options
+
+In Deno ecosystem, all strict flags are enabled in order to comply with TypeScript
+ideal of being `strict` by default. However, in order to provide a way to support customization a configuration file such as `tsconfig.json` might be provided to Deno on
+program execution.
+
+You do need to explicitly tell Deno where to look for this configuration, in order to do so
+you can use the `-c` argument when executing your application.
+
+```bash
+deno -c tsconfig.json mod.ts
+```
+
+Currently allowed settings, as well as their default values in Deno go as follows:
```

Comment:
Might be good to link to https://www.typescriptlang.org/docs/handbook/compiler-options.html somewhere here.

Reply:
Solved

Reply:
Added